app/utils/utils.ts

```diff
@@ -34,6 +35,50 @@ export function validateNewRuleset(newRuleset: Ruleset, rulesets: Ruleset[]): {
 
   return { isValid: true };
 }
+export function validateHours(newHoursWorked: EnterHours, hoursWorked: EnterHours[]): { isValid: boolean, errorMessage?: string } {
+
+  if (newHoursWorked.startTime == newHoursWorked.endTime) {
+    return { isValid: false, errorMessage: 'Start time and end time cannot be the same.' };
+  }
+
+  if (newHoursWorked.startTime > newHoursWorked.endTime) {
+    return { isValid: false, errorMessage: 'End time cannot be before start time.' };
+  }
+
```

Comment:
There's likely a lot of overlap with these utils, should we consider consolidating them into one file and making these checks more reusable as they are essentially the same inputs what do you think?